perun-cli/Perun/beans/Sponsor.pm

```diff
@@ -0,0 +1,45 @@
+package Perun::beans::Sponsor;
+
+use strict;
+use warnings;
+
+use Perun::Common;
+
+sub new
+{
+	bless({});
+}
+
+sub fromHash
+{
+        return Perun::Common::fromHash(@_);
+}
+
+sub TO_JSON
+{
+	my $self = shift;
+
+	return { user => $self->{_user} };
+}
+
+sub getUserId {
+        return shift->{_user}->{id};
+}
+
+sub getValidytyTo {
+	return shift->{_validytyTo};
+}	
+
```

Comment:
Would be great to have more "get" method for listing sponsors (at least firstName and lastName).